sw/device/silicon_creator/lib/cert/asn1.c

```diff
@@ -269,6 +269,10 @@ rom_error_t asn1_bitstring_push_bit(asn1_bitstring_t *bitstring, bool bit) {
 }
 
 rom_error_t asn1_finish_bitstring(asn1_bitstring_t *bitstring) {
+  if (bitstring->used_bits > 8) {
+    bitstring->state = NULL;
+    return kErrorAsn1InvalidArgument;
```

Comment:
- I think the test should be `>= 8` and the `kErrorAsn1Internal` because normally the `asn1_bitstring_push_bit` keeps the following invariant true: `0 <= used_bit < 8` (it pushes a byte when we reach 8 bits).

Reply:
ah, good catch, thanks. fixed.